net/aircrack-ng/Makefile

```diff
@@ -51,19 +67,35 @@ define Package/airmon-ng/description
   Bash script designed to turn wireless cards into monitor mode.
 endef
 
-TARGET_CFLAGS += -std=gnu89
+CONFIGURE_ARGS += \
+	--disable-silent-rules \
+	--enable-shared \
+	--enable-static \
+	--with-experimental \
+	--with-libpcap-include=$(STAGING_DIR)/usr/include \
+	--with-libpcap-lib=$(STAGING_DIR)/usr/lib \
+	--without-opt \
```

Comment:
Make that optional

Reply:
Which part am I to make optional here?

Reply:
```--without-opt```

Reply:
No, I'm using it here to ensure that the project doesn't add `-O2`.

Reply:
It adds -O3 which makes sense in terms of performance (in some cases)

Like here: https://github.com/openwrt/openwrt/blob/master/package/libs/zlib/Makefile#L58

Reply:
It should only improve cracking speed; however, I figure size is most important here. If cracking is the goal, one should be at least using a laptop/desktop system.

Having said that, if you want this to be a tweakable - I will add it to the Config.in file...

Reply:
(Also, most speed-related items are delegated towards SIMD and/or the SSL library chosen - at least with the aircrack-ng binary.)

Reply:
As we're moving towards faster platforms with less space restrictions (ARMv7 and higher for instance) it makes sense to prepare software which benefits from O3.

Reply:
Gotcha.

I've added it; however, the user-specified CFLAGS override the ones the autoconf generates. Because of this, the `-O3` added is overridden by the `-Os` being sent in.